rubocop-performance.gemspec

```diff
@@ -31,7 +31,7 @@ Gem::Specification.new do |s|
     'bug_tracker_uri' => 'https://github.com/rubocop-hq/rubocop-performance/issues'
   }
 
-  s.add_runtime_dependency('rubocop', '>= 0.90.0')
-  s.add_runtime_dependency('rubocop-ast', '>= 0.4.0')
-  s.add_development_dependency('simplecov')
+  s.add_runtime_dependency('rubocop', '~> 0.93.1')
+  s.add_development_dependency('rubocop-rspec', '~> 1.44')
+  s.add_development_dependency('simplecov', '~> 0.19.1')
```

Comment:
SimpleCov requires Ruby 2.5 or higher, but rubocop-performance supports for Ruby 2.4. So, the gem takes support for Ruby 2.4.

```console
#!/bin/bash -eo pipefail
bundle install
Fetching gem metadata from https://rubygems.org/.........
Resolving dependencies...
Bundler could not find compatible versions for gem "ruby":
  In Gemfile:
    ruby

    rubocop-performance was resolved to 1.8.1, which depends on
      ruby (>= 2.4.0)

    simplecov (~> 0.19.1) was resolved to 0.19.1, which depends on
      ruby (>= 2.5.0)

Exited with code exit status 6
CircleCI received exit code 6
```

Reply:
> but rubocop-performance supports for Ruby 2.4

For what, if Ruby 2.4 is no more maintained even for security releases?

https://www.ruby-lang.org/en/news/2020/04/05/support-of-ruby-2-4-has-ended/

Reply:
Using EOL Ruby 2.4 is (gem, application) users choice (of course, not recommended). It will be maintained for one year after EOL for migration. This is a maintenance policy and I will update a doc later.

Reply:
> Using EOL Ruby 2.4 is (gem, application) users choice (of course, not recommended). It will be maintained for one year after EOL for migration. This is a maintenance policy and I will update a doc later.

OK, thank you. I've reverted this and added some documentation right into `gemspec` file above `required_ruby_version`.